CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
 ### Fixes
 
 - Serialization on models where there's at least one boolean getter no longer fails.
+- Return type of `createSimpleSearchOperation ` becomes `ODataValueContextOfIListOfEntry` to contain more OData information compared to the previous `ODataValueOfIListOfEntry` class.
```

Comment:
Isn't it more accurate to say becomes CompletableFuture<ODataValueContextOfIListOfEntry> .... the previous CompletableFuture<ODataValueOfIListOfEntry> ?